jwst/pipeline/extract_1d.cfg

```diff
@@ -3,3 +3,4 @@ class = "jwst.extract_1d.Extract1dStep"
 
 log_increment = 50
 smoothing_length = 0
+subtract_background = None
```

Comment:
This might be the offending item causing the test failure.  `None` is special with ConfigObj.  If you pass it `None` in a parameter, it will read it as a string.  So our example configs should not have anything assigned to `None` in them.

Reply:
You are right that None can only be used as a default value in the spec. But I would have expected a ConfigObj validation error, not this bizarre failure.

Reply:
I would have as well.  If you remove that line, does it make the error go away for that test?  I recall seeing something like this when I added the `MasterBackgroundStep` and made the same mistake of putting an optional parameter set to `None` in the example .cfg file.

Reply:
Hmm, but maybe there's something a little more subtle going on.  I believe @stscieisenhamer originally wrote this suffix test.  It looks like it is getting this "new" suffix from the example .cfg file `name` entry.  But it should have been getting it before.  I don't know why it was not getting before and now it is.  Hmm.

Reply:
Yeah, there's definitely some subtle interaction going on.  If I you remove the
```
subtract_background = None
```
line from the config file, then the tests pass.  Hmm.  Would be nice to understand that.